src/components/Store.js

```diff
@@ -13,9 +14,32 @@ export function Store(props) {
       const child = document.querySelector(".nfd-feature-upsell > div");  
       child.classList.remove("nfd-grayscale") 
       document.querySelector(".nfd-feature-upsell .nfd-absolute.nfd-justify-center").style.backgroundColor = "rgba(255, 255, 255, .5)"
-    }    
+    }  
+
+    const storeContainer = document.querySelector('#ecommerce-features-wrapper');
+
+    const storeButtons = Array.from(
+      storeContainer.querySelectorAll( 'button' )
+    );
+    const storeAnchors = Array.from( storeContainer.querySelectorAll( 'a' ) );
+
+    if ( storeButtons.length ) {
+      storeButtons.forEach( ( button ) => {
+        button.addEventListener( 'click', onButtonNavigate );
+        button.addEventListener( 'onkeydown', onButtonNavigate );      
+      } );
+    }
+
+    if ( storeAnchors.length ) {
+      storeAnchors.forEach( ( link ) => {
+          link.addEventListener( 'click', onAnchorNavigate );
+          link.addEventListener( 'onkeydown', onAnchorNavigate );      
+      } );
+    }
+  
   }, []);
```

Comment:
We need to move the event triggers to the individual buttons and trigger them from the `onClick` prop.
Same for the 'View Analytics' anchor.